### ISSUE TITLE
Internal Error Has Occurred on fresh Ubuntu 18.04 LTS install

### ISSUE THREAD

**Issue by kaskr3x** [MISP/MISP#4948]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | Ubuntu 18.04 LTS
| MISP version / git hash   | 2.4
| Browser                   | Chrome

### Expected behavior
Being able to login 

### Actual behavior
Stuck at internal error has occurred with a link to download GnuPG key.

### Steps to reproduce the behavior
Fresh install; all I ran were the commands stated in the install guide. Installer ran all the way through, no problem, and I'm able to access Viper Web Interface & MISP Dashboard. Just can't access login page.

### Logs, screenshots, configuration dump, ...
![Screenshot 2019-08-02 at 12 04 06 AM](https://user-images.githubusercontent.com/50896359/62309041-3c972a80-b4b9-11e9-9346-46eebd32a37e.png)

**Comment by dmolina213** [comment]
It appears you were able to successfully get your dashboard going. I am having difficulty getting my dashboard upone seems to be have the time to help. Anyways this are the issues i opened https://github.com/MISP/misp-dashboard/issues/120
and https://github.com/MISP/misp-dashboard/issues/119

So other than the instructions provided in github. do you have any things you did above and beyond that is documented in the instructions? I would be eager to know and try it out in our instance.

Otherwise , I do apologize if this does not directly answer your question or issue?

v/r Doug Molina

**Comment by kaskr3x** [comment]
Yeah, I'm pretty sure I got everything up, with that one particular exception. I didn't encounter any other issues (other than the install portion being stuck at 56% progress, but it was easily fixed by running `sudo apt install haveged -qy` and `sudo service haveged start`). 

I went out on a limb here & tried regenerating the gnupg key (ref: #2372, combination of Gumnos's solution to the permission denied error whenever anyone tries to regenerate the key, along with iglocska's solution.), and did so successfully, but that didn't work out either, so I'm really clueless as to what's going on. 

The error logs (`/var/www/MISP/app/tmp/logs` directory, `debug.log` & `error.log`) weren't the most helpful either; I couldn't get any concrete errors out of them, though I honestly wouldn't know where to debug even if I got one concrete error that pointed me somewhere. 

This is the first time I'm deploying MISP with errors; previous builds worked fine for me so I really don't know what I'm doing wrong here.

Also, no worries about it, that's completely cool with me. I see that you're experiencing weird behaviour on the Docker instance?
